### ISSUE TITLE
loader run twice when using extract-text-webpack-plugin

### ISSUE THREAD

**Issue by MwumLi** [webpack-contrib/extract-text-webpack-plugin#259]
when I use  webpack with `extract-text-webpack-plugin` , I find loaders run twice
my webpack config like: 

``` javascript
    module: {
        loaders: [
          {
            test: /\.scss$/,
            loader: ExtractTextWebpackPlugin.extract("style-loader", "css!postcss!sass")
            //loader: "style-loader!css!postcss!sass"
          }        ]
    },
   postcss: function() {
     console.log('postcss ...');
     return [autoprefixer]; 
  }
```

when I run `webpack`, It print `postcss...` twice

**Comment by samuelpaulc** [comment]
@sokra 
I am also facing this issue. Any solution for this?

**Comment by bebraw** [comment]
Can you try upgrading to webpack 2 and see how it goes?

**Comment by pawelpabich** [comment]
Some possible workarounds can be found here: https://github.com/webpack/extract-text-webpack-plugin/issues/217

**Comment by michael-ciniawsky** [comment]
@MwumLi Please try again with webpack 2 / EWTP 2 and post a full error log if still regressions

**Comment by evilebottnawi** [comment]
Closing due to inactivity. Please test with latest version and feel free to reopen if still regressions. Thanks!

**Comment by milewski** [comment]
Im developing a postcss plugin and i confirm that this is still happening...

```
"extract-text-webpack-plugin": "3.0.1",
"webpack": "3.6.0"
```

and adding `allChunks: true` to the options seems to make it run only once....

**Comment by evilebottnawi** [comment]
@milewski can you create minimum reproducible test repo?

**Comment by milewski** [comment]
@evilebottnawi here you are `https://github.com/milewski/extract-text-webpack-plugin-bug` i made some copy and paste from my actual plugin.. and tried to simplified as much as i could....

run `npm run try` to see it in action...

**Comment by stereokai** [comment]
Please reopen. This bug still happens.

**Comment by d3viant0ne** [comment]
@milewski - If you don't mind, can you stick your comment along with your repo to reproduce into a new issue so this isn't buried back behind a years worth of issues & I'll get it properly labeled.

**Comment by stereokai** [comment]
@d3viant0ne Thanks. I am new to Webpack and using the latest version of all loaders. I am not doing anything out of the ordinary and still getting double output in my console when I am using ExtractTextPlugin. I hope this will be resolved soon. If I can do anything to help find out where the issue lies, please let me know

**Comment by d3viant0ne** [comment]
@stereokai - Let's save the comments until we have a fresh issue so we don't lose any current issue context.

**Comment by milewski** [comment]
okay #647
